docs/chainlink-keepers/introduction.md

```diff
@@ -15,34 +15,39 @@ whatsnext:
 
 ![Chainlink Keeper Network Banner](/images/contract-devs/generic-banner.png)
 
-**Chainlink Keepers** enables conditional execution of your smart contracts functions through a hyper-reliable and decentralized automation platform that uses the same external network of node operators that secures billions in value. Building on Chainlink Keepers will help you get to market faster so you don't have to deal with the setup cost, ongoing maintenance, and risks associated with a centralized automation stack. To take full advantage of the Keepers automation infrastructure, read all of the documentation to understand the features of Chainlink Keepers.
+**Chainlink Keepers** enable conditional execution of your smart contracts functions through a hyper-reliable and decentralized automation platform that uses the same external network of node operators that secures billions in value. Building on Chainlink Keepers will help you get to market faster so you don't have to deal with the setup cost, ongoing maintenance, and risks associated with a centralized automation stack. To take full advantage of the Keepers automation infrastructure, read all of the documentation to understand the features of Chainlink Keepers.
 
 To learn more about how the Chainlink Keepers Network automates your smart contracts, read the [Chainlink Keepers Architecture](../overview) page.
 
+<p>
+https://youtu.be/xL96sTwQ5Ho
+</p>
+
 **Topics**
 
-- [Select a Trigger](#select-a-trigger)
-  - [Time-based Trigger](#time-based-trigger)
-  - [Custom logic Trigger](#custom-logic-trigger)
-- [Supported Networks and Costs](#supported-networks-and-costs)
-- [Questions and Examples](#questions-and-examples)
+- [Select a trigger](#select-a-trigger)
+  - [Time-based trigger](#time-based-trigger)
+  - [Custom logic trigger](#custom-logic-trigger)
+- [Supported networks and costs](#supported-networks-and-costs)
+- [Questions and examples](#questions-and-examples)
```

Comment:
Let's stick with `-` characters for unordered lists. This is the default for the Prettier format as well.